query-engine/driver-adapters/src/proxy.rs

```diff
@@ -92,69 +92,78 @@ pub enum ColumnType {
     /// - INT16 (SMALLINT) -> e.g. `32767`
     /// - INT24 (MEDIUMINT) -> e.g. `8388607`
     /// - INT32 (INT) -> e.g. `2147483647`
-    Int32,
+    Int32 = 0,
```

Comment:
Nice that this is now explicit, so it's easier to see that it matches the the values in `const.ts`